recipes/libraw/all/CMakeLists.txt

```diff
@@ -26,13 +26,32 @@ if(WIN32)
     endif()
 endif()
 
-add_definitions(-DUSE_JPEG -DUSE_JPEG8 -DUSE_JASPER)
+if (TARGET CONAN_PKG::libjpeg OR TARGET CONAN_PKG::libjpeg-turbo)
+    add_definitions(-DUSE_JPEG -DUSE_JPEG8)
```

Comment:
https://github.com/LibRaw/LibRaw/blob/1dbed6b7e65ef2ebd0c3f82722a0b7f3990845f7/samples/mem_image_sample.cpp#L27

Do these need to be exposed to the customer?

Reply:
Interesting find. It seems that the defines are only used to write the sample program output in jpeg etc. outside of libraw functionality.

The [libraw.pc.in](https://github.com/LibRaw/LibRaw/blob/1dbed6b7e65ef2ebd0c3f82722a0b7f3990845f7/libraw.pc.in) does not have any `-DUSE_*` `CFLAGS` so I think we do not need to expose them in public interface.

Reply:
Awesome thanks for confirming